frontend/components/common.spec.js

```diff
@@ -37,7 +37,7 @@ describe('Common modules', function() {
 
     it('contains a link to the image', function() {
       const wrapper = render(<common.Badge src="/badge/foo-bar-blue.svg" />)
-      expect(wrapper.html()).to.contain('<img src="/badge/foo-bar-blue.svg"')
```

Comment:
Was the inclusion of the `img` tag unnecessary before, or if not, what's the thinking behind removing it (asking mostly for my own education on testing in React)

Reply:
Before I think it was `<img src="/badge…" alt="…">` and now I  think it's flipped around and turned into `<img alt="..." src="/badge...">` so the old version won't match anymore. This is now a really confusing assertion; why don't I figure out the full source of the `<img>` tag and use that.

Reply:
👍